src/default-rules/circular-dependencies/folder-dependency-graph.ts

```diff
@@ -71,6 +71,10 @@ export default (fileDependencyGraph: Graph) => {
 
         parentFolders.pop(); // ditch common root
         parentFolders.forEach(parent => {
+            if (registries.some(entry => parent.includes(entry))) {
```

Comment:
Which parents is it going to exclude?
I have three concerns here:
1. We should still be checking for circular references within the registries
2. The parent matching should be os-independent
3. According to my understanding, `parent.includes` is string matching and it could potentially result in incorrect matches e.g. `'foobar'.includes('foo')`

Reply:
1. Fixed,
2. Using micromatch now,
3. Micromatch should take care of it